style.css

```diff
@@ -129,3 +127,223 @@ header {
 #social-icons > a:hover {
   cursor: pointer;
 }
+
+#main-button {
+  position: relative;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+main {
+  background-color: #fff;
+  width: 100%;
+}
+
+ul {
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-inline-start: 40px;
+}
+
+#works {
+  display: grid;
+  grid-template-rows: 0.5fr repeat(3, 100%) 0.5fr;
+  grid-template-columns: auto;
+  grid-gap: 3%;
+  padding: 45px 24px;
+}
+
+#works-heading-wrap {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10% 5% 25%;
+}
+
+#works-heading {
+  font-weight: 700;
+  font-size: 44px;
+  line-height: 52px;
+  text-align: center;
+  width: 100%;
+  color: var(--color-primary);
+  padding: 0 0 45px 0;
+}
+
+#indicator {
+  width: 64px;
+  height: 4px;
+  background-color: var(--color-secondary);
+  border-radius: 24px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.work {
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-image {
+  margin: 0;
+  width: 100%;
+}
+
+.work-title {
+  margin: 0;
+  font-size: 27px;
+  line-height: 39px;
+  padding: 25px;
+  text-align: center;
+}
+
+.language-tags {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tag {
+  background: #ebf0ee;
+  border-radius: 4px;
+  font-size: 14px;
+  font-weight: 500;
+  padding: 5px;
+}
+
+.work-link {
```

Comment:
You really worked hard to implement all the project requirements :clap:. There is just this requirement that you need to work on.

- [ ] The button "See project"  should update its state on hover Hover and Pressed (see figma  design) :arrow_down: 
![hover](https://user-images.githubusercontent.com/26736582/198751533-90f0157c-ab6c-49f5-a652-f24764d799e3.png)